changelog.md

```diff
@@ -5,10 +5,14 @@
 **Fixed:**
 - react-native-bpk-component-text-input:
   - Made hit area larger to make activation of the Text Input easier on a touch-device
+
 **Added:**
 - react-native-bpk-component-card
   - New component BpkCard
 
+- bpk-tokens
+  - New `Color` class for setting opacity of Bpk colours
```

Comment:
This is incorrect, you have actually added `setOpacity` ;)

Suggestion:
```
- Color tokens can now be made opaque with `setOpacity`
```

Reply:
Color should be `setOpacity`

Reply:
Sorry this has been picked up already, ignore me 😄